modules/server/src/main/scala/higherkindness/compendium/Main.scala

```diff
@@ -17,7 +17,7 @@
 package higherkindness.compendium
 
 import cats.effect._
-import cats.syntax.functor._
+import cats.implicits._
```

Comment:
This is a good topic of discussion, whether to use all import vs specific imports 🙂

Reply:
I'm good with both approaches, IIRC @JesusMtnez in a previous review suggested changing multiple `syntax/instances` imports for a single one bringing everything to scope. Maybe this is not the case and I should keep that single `syntax` import, what do you think?

Reply:
Well, in general I don't like the idea of imports stuffs that I don't need in my code. This feeling is even worse when this extra unnecessary imports are bringing me tons of implicits. 

The package `cats.implicits._` bring a lot of magic inside it, and it is true, it is really confortable just type it and forget about everything, but I like to be more specific about what implicits I am bringing to my scope and why I am doing it

Reply:
Well, I like the explicit imports because I can see what is in use and what isn't, aside from avoiding partially the autocompletion madness caused by bringing all `syntax` modules into scope. But it's only a matter of preference that has nothing to do with actual code implementation or performance. As I said, I'm happy using `all` imports as this seems to be a common practice in these repositories and I feel that code should look homogeneous.

Reply:
I agree with @tzimisce012 I prefer to add each import for each implicit I need, most of all to avoid conflicts, but for some people, especially the newbies, it's more comfortable to use `cats.implicits._`. I think is also useful to remember that you can get all the syntax implicits using `cats.syntax.all._` leaving aside the instances implicits. That still adds a half-ton of implicits but keep the imports smaller when you need 10 syntax imports (and you can always delete it when you have conflicts)